packages/cf-deploy-config-sub-generator/src/app/index.ts

```diff
@@ -45,6 +45,7 @@ import {
 } from '@sap-ux/cf-deploy-config-inquirer';
 import type { YeomanEnvironment } from '@sap-ux/fiori-generator-shared';
 import type { Answers } from 'inquirer';
+import { readJSON } from '@sap-ux/project-access/dist/file';
```

Comment:
Is deep import intentional here? If yes please add a comment.

Reply:
Thanks @devinea! Updated now.